Weekly Changelog 2024-05-27 to 2024-06-03 (#74274)
CleverRaven/Cataclysm-DDA: data/changelog.txt

Graphical construction preview
Resume butchering
The player can swap between defined outfits with the 'set of clothes' item

> Add bionic multimeter and bench electronics kit
> Add scrapple as a comestible
> Allow ammo effects to cast spells
> Automatic turrets will not fire guns if they are going to overheat
> Unholy Debug Angel Cake added to the game. 10k calories for easy calories gain and for basecamp companions
> Hold your arrow

Reviewer: ```suggestion
Allow ammo effects to cast spells
Automatic turrets will not fire guns if they are going to overheat
```

Add three new locations to Aftershock
New jewelry added
Night vision helmet attachments

> Add weighted blankets and variants for them and for regular blankets.
> fish emulsion
> Adds Taurus Judge
> Updated terminology in lore doc.
> Static NPC factions have their own camps and food stores
> Adds one variant of Habitat Block to the Aftershock Exoplanet. Also adds hackable display cases and doors.
> Adds the CP33
> Regen serum
> Vitamin Cost per mutation
> Ruger MK IV
> More tie-able fridges/freezers

Reviewer: ```suggestion
fish emulsion
Static NPC factions have their own camps and food stores
Regen serum
Vitamin Cost per mutation
```

[Xedra Evolved] Make Paraclesian fae's magic easier in appropriate terrain
Aftershock: Basic Hacking EOCs
[Xedra Evolved] Make Paraclesian fae's magic harder in some circumstances

> Aftershock: Fix several low priority issues.
> Psionic powers say "Power level" instead of "Spell level" in the UI
> [MoM] Update chance to bypass prerequisites when learning psionic powers to use `test_eoc`
> [Xedra Evolved] Update Homullus civilization checker to run once per OMT
> [Xedra Evolved] Update Undine water-checking EoCs to use `test_eoc`
> [Xedra Evolved] Update Arvore forest-checking EoCs to use `test_eoc`
> [MoM] Torrential channeling research no longer has required proficiencies
> Aftershock: Add urban farms as town buildings
> Summon Mojocycle gets better at lvl 10
> [Magiclysm] Add more enchants to effect scaling

Reviewer: ```suggestion
Aftershock: Adds one variant of Habitat Block to the Aftershock Exoplanet. Also adds hackable display cases and doors.
Psionic powers say "Power level" instead of "Spell level" in the UI
[MoM] Torrential channeling research no longer has required proficiencies
Aftershock: Add urban farms as town buildings
Summon Mojocycle gets better at lvl 10
```

Make Skunks Less Bloodthirsty
Oops - remove cramped space when no longer cramped
Correctly display the cost to learn the proficiency

> Fixes a grammatical error within the cataclysm poem. It is now perfect.
> Correctly display pain penalties.
> Change coverage for heavy survivor boots and gloves
> make wood rails flammable
> Make salvaged exoskeletons actually non functional
> Adjust P50 Reload Time
> [Magiclysm] Reduce preserved food in druid tower
> Remove 9mm from .22 spawns
> Fix inconsistencies in corn and popcorn
> Fix Various Typos/Mistakes in Flier Snippets
> Added a weapon category to the G80
> fix sort armor ui list indexing
> Fix an assertion failure
> Fix Foam Guns and Hazardous foam not spawning on shelves in Aftershock Urban Farms.
> Fix cata-static-string_id-constants in char_volume_test
> Readjust descriptions for fridges/freezers
> Removes Railgun Fouling

Reviewer: ```suggestion
Correctly display pain penalties.
Removes Railgun Fouling
```
